.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,6 @@ Before we can merge this PR, please make sure that all the following items have
 checked off. If any of the checklist items are not applicable, please leave them but
 write a little note why.
 
-- [ ] Targeted PR against correct branch (`manta` or `dolphin`) with right title (start with [Manta] or [Dolphin]),
```

Comment:
I thought we used this convention to simply specify the merging branch, not where the changes are.
What do you guys think ? @stechu @Dengjianping

Reply:
I am all for drop it. Just for better ergonomics.

Reply:
ok true, as in the case of my #358 there is no value to leading it with manta, as it actually only affects Dolphin.

If y'all don't think seeing in the release notes https://github.com/Manta-Network/Manta/releases/tag/v3.1.4 which RT was affected by which PR, then yeah feel free to drop it. My proposal would be to change 

> [manta] Add a congested_chain_simulation test in Calamari by @ghzlatarev https://github.com/Manta-Network/Manta/pull/337

to

> [Calamari] Add a congested_chain_simulation test by @ghzlatarev https://github.com/Manta-Network/Manta/pull/337

or we could invent little shorthands like
MA/CA/DO + ALL if verbosity is the concern

Reply:
The reason we had this before is that we used to have multiple branches. It turns out to be very bad for both productivity and maintenance. Reducing complexity in any part of engineering at no other cost is always preferred.

Reply:
My idea here was to substitute the useless tag with one that actually gives valuable insight, instead of simply removing it.
I'm fine with it being gone too tho